examples/browser-to-server/tests/test.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable no-console */
+import { test, expect } from '@playwright/test'
+import { playwright } from 'test-util-ipfs-example'
+import { spawn, exec } from 'child_process'
+import { existsSync } from 'fs'
+
+// Setup
+const play = test.extend({
+  ...playwright.servers()
+})
+
+async function spawnGoLibp2p() {
+  if (!existsSync('../../examples/go-libp2p-server/go-libp2p-server')) {
+    await new Promise((resolve, reject) => {
+      exec('go build',
+        { cwd: '../../examples/go-libp2p-server' },
+        (error, stdout, stderr) => {
+          if (error) {
+            throw (`exec error: ${error}`)
+          }
+          resolve()
+        })
+    })
+  }
+
+  const server = spawn('./go-libp2p-server', [], { cwd: '../../examples/go-libp2p-server', killSignal: 'SIGINT' })
+  server.stderr.on('data', (data) => {
+    console.log(`stderr: ${data}`, typeof data)
+  })
+  const serverAddr = await (new Promise(resolve => {
+    server.stdout.on('data', (data) => {
+      console.log(`stdout: ${data}`, typeof data)
+      const addr = String(data).match(/p2p addr:  ([^\s]*)/)
+      if (addr !== null && addr.length > 0) {
+        resolve(addr[1])
+      }        
+    })
+  }))
+  return { server, serverAddr }
+}
+
+play.describe('bundle ipfs with parceljs:', () => {
+  // DOM
+  const connectBtn = '#connect'
+  const connectAddr = '#peer'
+  const messageInput = '#message'
+  const sendBtn = '#send'
+  const output = '#output'
+
+  let server
+  let serverAddr
+
+  // eslint-disable-next-line no-empty-pattern
+  play.beforeAll(async ({ }, testInfo) => {
+    testInfo.setTimeout(5 * 60_000)
+    const s = await spawnGoLibp2p()
+    server = s.server
```

Comment:
dunno what the convention is but unpacking `{ server, serverAddress} = s` should work here, no?

Reply:
No. This is assigning to closured variables.

Reply:
Hmmmm, ok.

Reply:
Yes, these are singletons where the variables are scoped in the `play.describe` closure.  The styleguide for this repo doesn't use semicolons, so doing the recommended destructuging assignment on an already defined variable causes issues:
```
({ server, serverAddr } = await spawnGoLibp2p())
```
The parser appends this to the `setTimeout()` function call above it (even with an extra line break), resulting in a TypeError: `TypeError: testInfo.setTimeout(...) is not a function`.

Adding semicolons resolves the issue but makes the code incompatible with the main `js-libp2p` styles and one that Aegir expects in lints.

I'm going to leave as is.

Reply:
More then fair enough. Sorry for the trouble 😰